ert/data/record/_record.py

```diff
@@ -332,9 +332,20 @@ async def load_collection_from_file(
     mime: str,
     length: int = 1,
     is_directory: bool = False,
+    smry_keys: Optional[List[str]] = None
 ) -> RecordCollection:
     if mime == "application/octet-stream":
-        if is_directory:
+        if smry_keys:
```

Comment:
Seems unnecessary at this time to instantiate a collection in each branch of only the data varies?